.github/workflows/main.yml

```diff
@@ -9,15 +9,15 @@ jobs:
         os:
           - macos-11
         otp:
-          - 24
```

Comment:
we want to keep 24, is it still possible?

Reply:
Sure. Brought it back.